src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -177,4 +189,36 @@ public with sharing class STG_PanelRD_CTRL extends STG_Panel {
         private set;
     }
 
+    public List<SelectOption> listInstallmentCreationOptions {
```

Comment:
By Apex standard, field names do not (need to) include the data type. For example, this field can be renamed to `installmentOppCreateOptions`.